### PR TITLE
Adding documentation for installation with conda.

### PR DESCRIPTION
**Pull request by strets123**

### CODE REVIEW

File: Docs/Book/Install.md [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+
+The easiest way to get Conda is having it installed as part of the [Anaconda Python distribution](http://docs.continuum.io/anaconda/install.html). A possible (but a bit more complex to use) alternative is provided with the smaller and more self-contained [Miniconda](http://conda.pydata.org/miniconda.html). The conda source code repository is available on [github](https://github.com/conda) and additional documentation is provided by the project [website](http://conda.pydata.org/). 
+
+### How to install RDKit with Conda
+
+Creating a new conda environment with the RDKit installed using these  packages requires one single command similar to the following::
+
+  $ conda create -c <channel-url> -n my-rdkit-env rdkit
+
+where '<channel-url>' is to be replaced with the URL of the package repository where the packages have been placed for distribution.
```
Comment (rvianello) [inline Docs/Book/Install.md]: what about replacing the <channel-url> placeholder with the actual rdkit channel on anaconda.org (or providing an example using this channel as a reference)?

File: Docs/Book/Install.md [hunk 2]
```diff
@@ -2,6 +2,61 @@
...
+
+  C:\> activate my-rdkit-env
+
+### How to build from source with Conda
+
+For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit)
+
+### Installing and using the RDKit Postgresql cartridge from a conda environment
+
+Due to the conda python distribution being a different version to the system python, it is easiest to install postgres and the postgres client via conda.
```
Comment (rvianello) [inline Docs/Book/Install.md]: was this meant to read 'postgres and the postgres cartridge'?

**Comment by rvianello** [comment]
thanks a lot for adding this paragraph, I added a couple of comments about minor details, but they can be also addressed at a later time.

**Comment by strets123** [comment]
Hi Riccardo, no problem, all the comments made sense so I have made some improvements to those sections.

**Comment by greglandrum** [comment]
Thanks to all who contributed to this!

**Pull request merged**